yarn.lock

```diff
@@ -5,91 +5,109 @@
 JSON2@0.1.0:
   version "0.1.0"
   resolved "https://registry.yarnpkg.com/JSON2/-/JSON2-0.1.0.tgz#8d7493040a63d5835af75f47decb83ab6c8c0790"
+  integrity sha1-jXSTBApj1YNa919H3suDq2yMB5A=
```

Comment:
can you undo this change to the integrity? We still use the older version of Yarn here. Thanks!